hadoop-master1/files/etc/ranger/conf/install.properties

```diff
@@ -0,0 +1,289 @@
+#SQL_CONNECTOR_JAR=/usr/share/java/postgresql.jar
+#SQL_CONNECTOR_JAR=/usr/share/java/sqljdbc4.jar
+#SQL_CONNECTOR_JAR=/opt/sqlanywhere17/java/sajdbc4.jar
+SQL_CONNECTOR_JAR=/opt/ranger/mysql-connector-j-__REPLACE_MYSQL_JDBC_VERSION__.jar
+
+
+#
+# DB password for the DB admin user-id
+# **************************************************************************
+# ** If the password is left empty or not-defined here,
+# ** it will try with blank password during installation process
+# **************************************************************************
+#
+#db_root_user=root|SYS|postgres|sa|dba
+#db_host=host:port              # for DB_FLAVOR=MYSQL|POSTGRES|SQLA|MSSQL       #for example: db_host=localhost:3306
+#db_host=host:port:SID          # for DB_FLAVOR=ORACLE                          #for SID example: db_host=localhost:1521:ORCL
+#db_host=host:port/ServiceName  # for DB_FLAVOR=ORACLE                          #for Service example: db_host=localhost:1521/XE
+db_root_user=root
+db_root_password=root
+db_host=mysql:3306
+#SSL config
+db_ssl_enabled=false
+db_ssl_required=false
+db_ssl_verifyServerCertificate=false
+#db_ssl_auth_type=1-way|2-way, where 1-way represents standard one way ssl authentication and 2-way represents mutual ssl authentication
+db_ssl_auth_type=2-way
+javax_net_ssl_keyStore=
+javax_net_ssl_keyStorePassword=
+javax_net_ssl_trustStore=
+javax_net_ssl_trustStorePassword=
+javax_net_ssl_trustStore_type=jks
+javax_net_ssl_keyStore_type=jks
+
+# For postgresql db
+db_ssl_certificate_file=
+
+#
+# DB UserId used for the Ranger schema
+#
+db_name=ranger
+db_user=root
+db_password=root
+
+#For over-riding the jdbc url.
+is_override_db_connection_string=false
+db_override_connection_string=
+
+
+# change password. Password for below mentioned users can be changed only once using this property.
+#PLEASE NOTE :: Password should be minimum 8 characters with min one alphabet and one numeric.
+rangerAdmin_password=Ranger@admin123
+rangerTagsync_password=Ranger@admin123
+rangerUsersync_password=Ranger@admin123
+keyadmin_password=Ranger@admin123
+
+
+#Source for Audit Store. Currently solr, elasticsearch and cloudwatch logs are supported.
+# * audit_store is solr
+audit_store=
+
+# * audit_solr_url Elasticsearch Host(s). E.g. 127.0.0.1
+audit_elasticsearch_urls=
+audit_elasticsearch_port=
+audit_elasticsearch_protocol=
+audit_elasticsearch_user=
+audit_elasticsearch_password=
+audit_elasticsearch_index=
+audit_elasticsearch_bootstrap_enabled=true
+
+
+# * audit_solr_url URL to Solr. E.g. http://<solr_host>:6083/solr/ranger_audits
+audit_solr_urls=
+audit_solr_user=
+audit_solr_password=
+audit_solr_zookeepers=
+
+audit_solr_collection_name=ranger_audits
+#solr Properties for cloud mode
+audit_solr_config_name=ranger_audits
+audit_solr_configset_location=
+audit_solr_no_shards=1
+audit_solr_no_replica=1
+audit_solr_max_shards_per_node=1
+audit_solr_acl_user_list_sasl=solr,infra-solr
+audit_solr_bootstrap_enabled=true
```

Comment:
ditto. and does audit support lightweight backend? for i.e. simply log the message

Reply:
the audit function of ranger admin is used to retrieve audit logs. Currently, it seems that only es and solr are supported, which needs further exploration

Reply:
it seems supports log4j

https://docs.cloudera.com/cdp-private-cloud-base/7.1.8/security-ranger-auditing/topics/security-ranger-audit-push-log-local-configs.html